src/plugins/git/render.test.js

```diff
@@ -2,15 +2,32 @@
 
 import * as GN from "./nodes";
 import {description} from "./render";
+import type {Repository} from "./types";
 
 describe("plugins/git/render", () => {
-  const examples = {
-    commit: (): GN.CommitAddress => ({
-      type: GN.COMMIT_TYPE,
-      hash: "3715ddfb8d4c4fd2a6f6af75488c82f84c92ec2f",
-    }),
-  };
+  const exampleHash = "3715ddfb8d4c4fd2a6f6af75488c82f84c92ec2f";
+  const exampleCommit: GN.CommitAddress = Object.freeze({
+    type: GN.COMMIT_TYPE,
+    hash: exampleHash,
+  });
+  const exampleRepository: Repository = Object.freeze({
+    commits: {
+      [exampleHash]: {
+        hash: exampleHash,
+        shortHash: exampleHash.slice(7),
+        summary: "This is an example commit",
+        parentHashes: [],
+      },
+    },
+  });
+
   it("commit snapshots as expected", () => {
-    expect(description(examples.commit())).toMatchSnapshot();
+    expect(description(exampleCommit, exampleRepository)).toMatchSnapshot();
```

Comment:
Not a product of this commit, but it seems a bit sketchy to be
snapshotting an arbitrary React node. No change requested—I’ll fix these
up in a separate PR.

Reply:
Well, never mind. Enzyme is still broken. This will have to do.